MLUtilities/Visualizations/Charting.py

```diff
@@ -96,7 +96,7 @@ def PlotSeriesWithErrorBars(seriesData, seriesErrorBars, seriesLabels, xAxisPoin
    fig, ax =__SetUpChart(chartTitle, xAxisTitle, yAxisTitle)
    
    for i in range(len(seriesData)):
-      ax.errorbar(xAxisPoints, seriesData[i], seriesErrorBars[i], marker= __GetMarker(i), color = __GetLineColor(i), linestyle=__GetLineStyle(i, useLines), label = seriesLabels[i])
+      ax.errorbar(xAxisPoints, seriesData[i], seriesErrorBars[i], marker= __GetMarker(i, True), color = __GetLineColor(i), linestyle=__GetLineStyle(i, useLines), label = seriesLabels[i])
```

Comment:
I'm going to move this to be a new default parameter on PlotSeriesWithErrorBars incase...someone wants to plot without markers...

But I'll do that in another commit - no need to adjust.